app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            pytest.param(14, 14, [0, 0],
+                         id="Up to 15 cat/dog years give 0 human year"),
+            pytest.param(15, 15, [1, 1],
+
+                         id="First 15 cat/dog years give 1 human year"),
+            pytest.param(23, 23, [1, 1],
+                         id="24 cat/dog years should be equal 1 human years"),
+            pytest.param(27, 27, [2, 2],
+                         id="Every 4 next cat years give 1 extra human year"),
+            pytest.param(27, 28, [2, 2],
+                         id="Every 5 next dog years give 1 extra human year"),
+            pytest.param(28, 28, [3, 2],
+                         id="28 cat years should be equal 3 human years"),
+            pytest.param(28, 29, [3, 3],
+                         id="29 dog years should be equal 3 human years"),
+        ],
```

Comment:
There are not all tests which I want to see. Please check the checklist before you send task on review 
![image](https://user-images.githubusercontent.com/104218550/200174843-90e7284b-49d0-49bd-9029-d097261f103d.png)